SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -22,9 +22,19 @@ struct FPendingRPCParams
 	SpatialGDK::RPCPayload Payload;
 };
 
-class FRPCContainer
+class SPATIALGDK_API FRPCContainer
 {
 public:
+	FRPCContainer() = default;
+
+	FRPCContainer(const FRPCContainer&) = delete;
+	FRPCContainer& operator=(const FRPCContainer&) = delete;
+
+	FRPCContainer(FRPCContainer&&) = default;
+	FRPCContainer& operator=(FRPCContainer&&) = default;
+
+	~FRPCContainer() = default;
+
 	void QueueRPC(FPendingRPCParamsPtr Params, ESchemaComponentType Type);
 	void ProcessRPCs(const FProcessRPCDelegate& FunctionToApply);
```

Comment:
Not actually part of this PR but why is processing stuff done like this instead of just making this a container?

Reply:
I agree, we could expose iterators for this custom container and do the processing externally, if that's what you're suggesting. But it's not part of this PR :)

Reply:
Do you even need iterators? It's basically a more complicated queue.

Reply:
Wouldn't we have to implement ProcessRPCs function externally then?

Reply:
Yes that's the point. I think it's odd that we provide a function to the container rather than putting RPCs in it and then taking them out.

Reply:
Would you mind if I resolve it since it's not a part of this PR?

Reply:
Sure